test/Npgsql.Tests/Types/ArrayTests.cs

```diff
@@ -132,7 +132,7 @@ public void EmptyArray()
         }
 
         [Test, Description("Verifies that the array type returned from NpgsqlDataReader.GetValue() is always compatible with null values.")]
-        public void GetValueArrayTypeForValueTypesDependsOnActualValue()
+        public void GetValueArrayTypeForValueTypesIsAlwaysNullable()
```

Comment:
Thanks for renaming, but maybe remove `Always` (since we now have non-null domains)

Reply:
Agreed.